pkg/actors/actors.go

```diff
@@ -228,15 +228,8 @@ func newActorsWithClock(opts ActorsOpts, clock clock.WithTicker) (ActorRuntime,
 
 	a.timers.SetExecuteTimerFn(a.executeTimer)
 
-	if opts.Config.SchedulerService != "" {
+	if opts.Config.SchedulerManager != nil {
 		log.Info("Using Scheduler service for reminders.")
-		// TODO: have a wrapper that includes both client and conn.
```

Comment:
We don't need this part anymore since the scheduler manager manages the next client conns. I still thought it was nice to have the log though.